git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -56,7 +56,7 @@ previous_sequential_module: git_setup_mac_and_linux, git_setup_windows
 @end
 
 @version_history 
-No previous versions.
+- [1.0.9](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/16a4a852199689a42f04555cb581cf2dcb90fb0f/git_creation_and_tracking/git_creation_and_tracking.md#1): Initial version
```

Comment:
```suggestion
- [1.0.10](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/16a4a852199689a42f04555cb581cf2dcb90fb0f/git_creation_and_tracking/git_creation_and_tracking.md#1): Initial version
```

"initial version" was revised up to 1.0.10 before we got around to merging this in.